src/sdk/TranslationRecognizer.ts

```diff
@@ -184,23 +183,17 @@ export class TranslationRecognizer extends Recognizer {
         try {
             Contracts.throwIfDisposed(this.privDisposedTranslationRecognizer);
 
-            this.implCloseExistingRecognizer();
-
-            this.privReco = this.implRecognizerSetup(
-                RecognitionMode.Conversation,
-                this.properties,
-                this.audioConfig,
-                new TranslationConnectionFactory());
+            this.implRecognizerStop();
 
             this.implRecognizerStart(
-                this.privReco,
+                RecognitionMode.Conversation,
```

Comment:
> RecognitionMode.Conversation, [](start = 15, length = 30)

can this be overridden by properties? (which means we should take it from the properties instead of hardcoding)